src/index.tsx

```diff
@@ -1,3 +1,6 @@
+import 'react-app-polyfill/ie11';
+import 'react-app-polyfill/stable';
```

Comment:
are these added just to make sure that ie11 works? or just for safety?

Reply:
Console shows `Unable to get property includes` and `import 'react-app-polyfill/stable';` adds polyfill for it. I retested this and `import 'react-app-polyfill/ie11';` is not needed and I will remove it

<img width="710" alt="Screenshot 2020-11-18 at 14 37 49" src="https://user-images.githubusercontent.com/24706814/99532153-8193e880-29ac-11eb-8415-d5bfd0b74bb2.png">